### ISSUE TITLE
CIFAR-10 fine-tuning hyperparameters

### ISSUE THREAD

**Issue by ganeshkumar5699** [google-research/vision_transformer#84]
We have been trying to reproduce the **98.95%** accuracy obtained by fine-tuning ViT_B16 on CIFAR-10. What hyperparameters were used for this result? Is it the same as the one on the official Colab? That is, base lr = 0.03, cosine lr decay (the same function in hyper.py), 100 epochs, no weight decay and a batch size of 512.

Since the paper does not mention all the hyperparameters for this, we have been relying on the Colab (but that seems to use different hyperparameters). 

The official Colab notebook notes that a **"~97.2%**" accuracy would be obtained when fine-tuning using the model/hyperparameters on Colab. What should we change in the Colab to obtain the accuracy in the paper?

Thanks for uploading the code for vision transformers :). This research project is pretty interesting.

**Comment by andsteing** [comment]
Hi ganeshkumar5699

The Colab only trains for a very short time, hence the low numbers.

Please refer to the main table in the `README.md` from the repository to see actual runs that reproduce the numbers from the paper:

https://github.com/google-research/vision_transformer

You'll notice that the two runs from the B/16 got to 98.92% and 99.03% after 7.2h of training on a 8x V100 machine. For the flag values you can refer to the "hparams" tab of the tensorboard.dev link from the table (for the B/16 only the `--accum_steps=16` was changed to fit on the V100's memory).

**Comment by ganeshkumar5699** [comment]
Thank you for the response. This was helpful.

**Comment by deepankarvarma** [comment]
@ganeshkumarashok Can you please share the notebook ? Thank you.
